src/components/shared/misc/SummarisedMultiSelector.vue

```diff
@@ -0,0 +1,225 @@
+            */
+            options: {
+                type: Array,
+                default() {
+                    return []
+                },
+            },
+
+            /**
+            * Maps the dropdown to the title/value fields on your options array.
+            */
+            fields: {
+                type: Object,
+                default() {
+                    return {
+                        title: 'name',
+                        value: 'id',
+                    }
+                },
+            },
+
+            /**
+            * The current value of the dropdown, v-model alias
+            */
+            value: {
+                required: true,
+                type: [String, Array],
+                default() {
+                    return ''
+                },
+            },
+
+            /**
+            * An optional placeholder for the dropdown
+            */
+            placeholder: {
+                type: String,
+                default() {
+                    return ''
+                },
+            },
+
+            /**
+            * Default text for the dropdown summary popup
+            */
+            defaultSummary: {
+                type: String,
+                default() {
+                    return 'Select an item'
+                },
+            },
+
+            /**
+            * Semantic ui dropdown configuration
+            */
+            dropdownSettings: {
+                type: Object,
+                default() {
+                    return {
+                        useLabels: false,
+                        showOnFocus: true,
+                        onHide: () => this.$emit('dropdown-hidden'),
+                    }
+                },
+            },
+
+            /**
+            * If the component should be in read only mode
+            */
+            readOnly: {
+                type: Boolean,
+                default() {
+                    return false
+                },
+            },
+
+            /**
+            * If the component should be inline
+            */
+            inline: {
+                type: Boolean,
+                default() {
+                    return true
+                },
+            },
+
+            /**
+            * If the component should be fluid
+            */
+            fluid: {
+                type: Boolean,
+                default() {
+                    return true
+                },
+            },
+        },
+
+        computed: {
+            model: {
+                get() {
+                    return this.value
+                },
+
+                set(val) {
+                    this.$emit('input', val)
+                },
+            },
+
+            summary() {
+                const summary = this.model && Array.isArray(this.model) ? this.model.join(', ') : this.model.split(',').join(', ')
+                if (summary.length) {
+                    return summary
```

Comment:
This still displays the `id` rather than the `name` of what is selected.
I have changed the styleguide mixin to use a more common data structure and it shows `1, 2, 3` instead of `one, two, three`
```js
basic: [
  { id: 1, name: 'one' },
  { id: 2, name: 'two' },
  { id: 3, name: 'three' },
],
```